CHANGELOG.md

```diff
@@ -1639,3 +1639,5 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#3778](https://github.com/open-telemetry/opentelemetry-python/pull/3778))
 - Fix license field in pyproject.toml files
   ([#3803](https://github.com/open-telemetry/opentelemetry-python/pull/3803))
+- bump trace-context to e99a85d12df8c25c51ec1d5d234f20e918265b75
+  ([#4109](https://github.com/open-telemetry/opentelemetry-python/pull/4109))
```

Comment:
You should move this from the "# Unreleased" section at the beginning of the CHANGELOG.md file